modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -201,9 +203,17 @@ func (suite *KeeperTestSuite) TestQueryIncentivizedPacketsForChannel() {
 						CountTotal: false,
 					},
 					PortId:      ibctesting.MockFeePort,
-					ChannelId:   ibctesting.FirstChannelID,
+					ChannelId:   path.EndpointA.ChannelID,
 					QueryHeight: 0,
 				}
+				identifiedFees1 := types.NewIdentifiedPacketFees(channeltypes.NewPacketID(ibctesting.MockFeePort, path.EndpointA.ChannelID, 1), packetFees.PacketFees)
+				identifiedFees2 := types.NewIdentifiedPacketFees(channeltypes.NewPacketID(ibctesting.MockFeePort, path.EndpointA.ChannelID, 2), packetFees.PacketFees)
+				identifiedFees3 := types.NewIdentifiedPacketFees(channeltypes.NewPacketID(ibctesting.MockFeePort, path.EndpointA.ChannelID, 3), packetFees.PacketFees)
+
+				expIdentifiedPacketFees = append([]*types.IdentifiedPacketFees{}, &identifiedFees1, &identifiedFees2, &identifiedFees3)
+				for _, identifiedPacketFees := range expIdentifiedPacketFees {
+					suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), identifiedPacketFees.PacketId, types.NewPacketFees(identifiedPacketFees.PacketFees))
+				}
```

Comment:
This is a bit ugly, but down in the main test logic we use `FirstChannelID` and we can't do it now as the channelID depends on the path being setup